### DISCUSSION TITLE
thoughts-on-success-and-failure/

### DISCUSSION THREAD

**Discussion by giscus** [Philo-Li/philo-li.github.io#3] [General]
# thoughts-on-success-and-failure/

脱离成功主义叙事我的字典里就没有成功和失败这俩词，我不会用其评判自己和别人。对我来说，人只有美与不美（审美意义上）。

https://www.philoli.com/thoughts-on-success-and-failure/

**Comment by WhyWhatHow** [comment]
收获良多,谢谢
